__tests__/app.test.js

```diff
@@ -70,10 +70,25 @@ describe("/api/reviews/:review_id", () => {
               category: expect.any(String),
               owner: expect.any(String),
               created_at: expect.any(String),
+              comment_count: expect.any(Number),
```

Comment:
you shouldn't be editing previous tests for new functionality. I'd expect this to be a new test.

Reply:
if you have you change an old test it's a clue that it was a brittle test